recipe-client-addon/test/browser/browser_PreferenceManagement.js

```diff
@@ -0,0 +1,166 @@
+"use strict";
+
+Cu.import("resource://gre/modules/Preferences.jsm", this);
+Cu.import("resource://gre/modules/TelemetryEnvironment.jsm", this);
+Cu.import("resource://shield-recipe-client/lib/PreferenceManagement.jsm", this);
+
+// Save ourselves some typing
+const Manager = PreferenceManagement("test");
+const withMockData = Manager.withMockData.bind(Manager);
+
+const DefaultPreferences = new Preferences({defaultBranch: true});
+
+function experimentFactory(attrs) {
```

Comment:
Can this function live in head.js instead of being duplicated between test files?